tests/requirements-tools.txt

```diff
@@ -3,3 +3,4 @@ pytest
 pytest-cov
 pytest-clarity
 pytest-subtests
+copier
```

Comment:
This is already in https://github.com/XaF/qolsysgw/blob/main/tests/end-to-end/requirements.txt (for end-to-end testing), this is not required for other tests!